site/docsource/Installation.adoc

```diff
@@ -16,7 +16,7 @@ is installed, run the following command:
 
 [source,sh]
 ------------------------------
-npm install --save bs-platform
+npm install --save-dev bs-platform
```

Comment:
Well I have no experience with Typescript but I'm not sure it's wise to push libraries on NPM without at least the JS build. Actually I would push both the compiled JS and the ML source. At least that's how we do with our babel based libraries.

Building during postinstall is nice, but it adds an overhead for both the bucklescript users and the non bucklescript users. Bucklescripts users already have the tools to compile the lib, and those who don't would prefer less dependencies to download.